lib/util/git/auth.ts

```diff
@@ -124,3 +127,87 @@ export function getAuthenticationRules(
 
   return authenticationRules;
 }
+
+export function getGitEnvironmentVariables(
```

Comment:
please create a seperate refactor PR for this to be merged before, so we've less changes here to review